tests/API/HcaApiTestAutomationFramework/HcaApiTestAutomationFramework/GraphQlClient.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System.Collections.Generic;
+using System.Configuration;
+using System.Net;
+using Braintree;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using RestSharp;
+
+namespace HcaApiTestAutomationFramework
+{
+	public class GraphQlClient
+	{
+		private RestClient _client;
+
+		public string getClientToken()
+		{
+			BraintreeGateway gateway = new BraintreeGateway
+			{
+				Environment = Braintree.Environment.SANDBOX,
+				MerchantId = ConfigurationManager.AppSettings["BraintreeMerchantId"],
+				PublicKey = ConfigurationManager.AppSettings["BraintreePublicKey"],
+				PrivateKey = ConfigurationManager.AppSettings["BraintreePrivateKey"],
+			};
+			gateway.Configuration.Timeout = 10000;
+			var clientToken = gateway.ClientToken.Generate();
+			return clientToken;
+		}
+
+		public string Base64Decoder(string stringText)
+		{
+			string base64Encoded = stringText;
+			string base64Decoded;
+			byte[] data = System.Convert.FromBase64String(base64Encoded);
```

Comment:
```suggestion
			var base64Encoded = stringText;
			var base64Decoded;
			var data = System.Convert.FromBase64String(base64Encoded);
```
why do you use explicit type declaration in variables?
- var requires less typing. var is shorter and easier to read, for instance, than Dictionary <int, IList <string>>.
- var requires less code changes if the return type of a method call changes. You only have to change the method declaration, not every place it’s used.
- var encourages you to use a descriptive name for the variable. This means the instance, not the type name. For instance:
- var customer = new Customer () rather than var c = new Customer ().

>